rerun_py/rerun_sdk/rerun/log_deprecated/rects.py

```diff
@@ -125,7 +125,7 @@ def log_rects(
 
     Colors should either be in 0-255 gamma space or in 0-1 linear space.
     Colors can be RGB or RGBA. You can supply no colors, one color,
-    or one color per point in a Nx3 or Nx4 numpy array.
+    or one color per point in a Nx3 or Nx4 numpy rects.
```

Comment:
the colors isn't an "numpy rect" (nothing says "that many arrays per rect", it's just a single array with the given dimension)

Reply:
sed gone wrong